pkgs/top-level/python-packages.nix

```diff
@@ -4377,6 +4377,8 @@ in {
 
   uritemplate = callPackage ../development/python-modules/uritemplate { };
 
+  weasyprint = callPackage ../development/python-modules/weasyprint { };
```

Comment:
Sorry I didn't find this earlier. This file should be ordered somewhat alphabetically.

Reply:
Thanks for taking a look, I've fixed that!